### DISCUSSION TITLE
虽然很不想问但是得问一下

### DISCUSSION THREAD

**Discussion by Haif-07** [anghunk/linuxdo-scripts#55] [Q&A] [answered]
我的为什么不能正常触发，或者我不知道怎么打开？
![image](https://github.com/user-attachments/assets/5eab026e-dca2-4ba1-a6a3-aa682380d48a)

**Comment by anghunk** [comment]
打开f12再刷新一下页面，看看有什么报错

**Reply by Haif-07** [reply]
![image](https://github.com/user-attachments/assets/b8da7b6b-9780-4473-b29e-64e840a0df7c)

**Reply by anghunk** [reply, marked as answer]
像是网络不好，没把那几个公共依赖cdn加载起来 😂
